packages/preset-mini/README.md

```diff
@@ -82,6 +82,38 @@ will generate:
 }
 ```
 
+### Theme
+You can fully customize your theme configuration, and UnoCSS will eventually deeply merge user configuration and default configuration.
+
+```ts
+presetMini({
+  theme: {
+    // ...
+    colors: {
+      primary: '#ff0000',
+    },
+  },
+})
+```
+
+Notably, unless `breakpoint` properties. UnoCSS gives `breakpoint` control to the user, you need to fully customize the `breakpoint` properties that you want to use. For example:
```

Comment:
```suggestion
One exception is that UnoCSS gives full control of `breakpoints` to users. When a custom `breakpoints` is provided, the default will be overridden instead of merging. For example:
```